Modules/mathmodule.c

```diff
@@ -3421,6 +3421,28 @@ math_ulp_impl(PyObject *module, double x)
     return x2 - x;
 }
 
+static int
+math_exec(PyObject *module) {
```

Comment:
PEP 7 recommands to put { on a separated line.

```suggestion
static int
math_exec(PyObject *module)
{
```